### DISCUSSION TITLE
win10下scoop安装报错。

### DISCUSSION THREAD

**Discussion by xydcz** [he3-app/he3-feedback#85] [Bug Report]
win10 20h2。scoop安装报错。
scoop install "https://raw.githubusercontent.com/he3-app/he3-scoop/main/he3.json"
Installing 'he3' (1.2.1) [64bit]
Checking hash of He3_windows_installer_x64_1.2.11.exe ... ok.
Running installer... error.
ERROR 使用“0”个参数调用“Start”时发生异常:“请求的操作需要提升。”
Installation aborted. You might need to run 'scoop uninstall he3' before trying again.

**Comment by hongchaodeng** [comment]
你好，能够提供更多重现的方式吗？比如你的 shell 是什么，环境怎么启动，运行了什么命令等？我们好复现一下问题。

**Reply by lyzhang1999** [reply]
需要管理员权限。

**Comment by lyzhang1999** [comment]
用管理员权限运行终端再安装即可。

**Reply by xydcz** [reply]
这样可以
